packages/cli/src/databases/entities/api-keys.ts

```diff
@@ -0,0 +1,20 @@
+import { Column, Entity, Index, ManyToOne } from '@n8n/typeorm';
+import { WithTimestampsAndStringId } from './abstract-entity';
+import { User } from './user';
+
+@Entity('user_api_keys')
+@Index(['userId', 'label'])
+export class ApiKeys extends WithTimestampsAndStringId {
+	@ManyToOne(() => User, (user) => user.id)
+	user: User;
+
+	@Column('string')
+	userId: string;
+
+	@Column({ type: String })
+	label: string;
+
+	@Index({ unique: true })
+	@Column({ type: String })
+	apiKey: string;
```

Comment:
@tomi should we do here on delete cascade? In the tests is very annoying to remember to delete the API keys table before deleting the user table, or you will get an foreign key error.

Reply:
Yes makes sense 👍